src/components/Nav/Nav.js

```diff
@@ -1,8 +1,108 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Nav.scss';
+
+class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      scrollTop: 0,
+      isLogin: false,
+      category: ['DINING', 'CAFE', 'CUPBOARD'],
+    };
+  }
+
+  handleScroll = e => {
+    const scrollTop = ('scroll', e.srcElement.scrollingElement.scrollTop);
+    this.setState({
+      scrollTop,
+    });
+  };
+
+  componentWillUnMount = () => {
+    window.removeEventListener('scroll', this.handleScroll);
+  };
+
+  componentDidMount = () => {
+    window.addEventListener('scroll', this.handleScroll);
+
+    if (!localStorage.getItem('userToken')) {
+      this.setState({
+        isLogin: false,
+      });
+    } else {
+      this.setState({
+        isLogin: true,
+      });
+    }
```

Comment:
필요한 부분에만 삼항연산자 활용해서 다음처럼 간략하게 쓸 수 있을 것 같습니다.

```suggestion
    this.setState({
      isLogin: localStorage.getItem('userToken') ? true : false,
    });
```